assets/js/composer.js

```diff
@@ -507,7 +507,7 @@ PageComposer.prototype = {
         data: $form.serialize(),
         type: method,
         headers: {
-          Accept: 'text/html, application/xhtml+xml;',
+          Accept: 'application/json',
```

Comment:
This fixes an error on edit block and clicking to save. (I fixed this on another place in this js, and forgot to fix this one aswell)

Reply:
Cant be really tested without pnther tests